src/main/java/com/xiaomi/infra/pegasus/rpc/async/TableHandler.java

```diff
@@ -230,7 +230,8 @@ public void run() {
                 onUpdateConfiguration(query_op);
               }
             },
-            5);
+            5,
+            getDefaultTimeout());
```

Comment:
But what if he set the default timeout too small? It means even if he initially opens the table at a long timeout, when the table configuration is changed he still has problem on opening table.